examples/multiple_windows/lib/app/regular_window_content.dart

```diff
@@ -0,0 +1,149 @@
+    )..repeat();
+    cubeColor = _generateRandomDarkColor();
+  }
+
+  @override
+  void dispose() {
+    _animation.dispose();
+    super.dispose();
+  }
+
+  Color _generateRandomDarkColor() {
+    final random = Random();
+    const int lowerBound = 32;
+    const int span = 160;
+    int red = lowerBound + random.nextInt(span);
+    int green = lowerBound + random.nextInt(span);
+    int blue = lowerBound + random.nextInt(span);
+    return Color.fromARGB(255, red, green, blue);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final dpr = MediaQuery.of(context).devicePixelRatio;
+    final windowSize = WindowScope.contentSizeOf(context);
+    final WindowManager windowManager = WindowManagerAccessor.of(context);
+    final WindowSettings windowSettings = WindowSettingsAccessor.of(context);
+
+    final child = Scaffold(
+      appBar: AppBar(title: Text('Regular Window')),
+      body: Center(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                RotatedWireCube(animation: _animation, cubeColor: cubeColor),
+              ],
+            ),
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                ElevatedButton(
+                  onPressed: () {
+                    final UniqueKey key = UniqueKey();
+                    windowManager.add(
+                      KeyedWindow(
+                        key: key,
+                        controller: RegularWindowController(
+                          preferredSize: windowSettings.regularSize,
+                          delegate: CallbackRegularWindowControllerDelegate(
+                            onDestroyed: () => windowManager.remove(key),
+                          ),
+                          title: 'Regular',
+                        ),
+                      ),
+                    );
+                  },
+                  child: const Text('Create Regular Window'),
+                ),
+                const SizedBox(height: 20),
+                Text(
+                  'View #${widget.window.rootView.viewId}\n'
+                  'Size: ${(windowSize.width).toStringAsFixed(1)}\u00D7${(windowSize.height).toStringAsFixed(1)}\n'
+                  'Device Pixel Ratio: $dpr',
+                  textAlign: TextAlign.center,
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+
+    return ViewAnchor(
+      view: ListenableBuilder(
+        listenable: windowManager,
+        builder: (BuildContext context, Widget? _) {
+          final List<Widget> childViews = <Widget>[];
+          for (final KeyedWindow window in windowManager.windows) {
+            if (window.parent == widget.window) {
```

Comment:
Just confirming: If I press "Create Regular Window" from one of the spinning cube windows, then that spinning cube window is the parent of the new window? I didn't realize there was a hierarchy like this when I first looked at the app because the table in the main window makes it look like every window is a sibling other than the main window.

Reply:
Oh I see, thanks. So this `if` will never evaluate to true until those new window types are implemented.